### ISSUE TITLE
Klimamessungen am Hauptturm

### ISSUE THREAD

**Issue by achristen** [envmet/ECOSENSE#4]
Es soll ein CR1000X basiertes System aufgebaut werden, welches folgende Grössen auf jeden Fall erfasst:

1. 4-Komponenten Strahlung (CNR4) auf Ausleger an der Turmspitze
2. PAR-Sensor (in and out) auf Ausleger an der Turmspitze
3. Beheizbare Niederschlagswippe auf Ausleger etwas über der Höhe der höchsten Bäume
4. CS125 Sichtweiten und Niederschlagsart-Messgerät ("Present Weather Sensor") auf Turmspitze
5. Profil von Temperatur / Feuchte Sensoren auf mindestens drei Höhen (2 m, Kronenraum, Turmspitze mit CS125 verbunden)
6. Profile von Temperatur / Feuchte Sensoren jeweils auf 2 m und im Kronenraum am Buchen- und Douglasienturm
7. Wind-Propeller auf Turmspitze
8. ClimaVue50 als Backup Gerät auf der Turmspitze (möglicherweise über separaten Logger oder über Profil-System-Logger da Redundanz?)

Zu diskutieren, ob auch folgende Geräte angehängt werden:

9. Möglicherweise Infrarot-Kameras (IRTs) auf den Canopy schauend von oben, bzw. auf den Waldboden
10. Sun-Tracker zur Erfassung der diffusen, direkten, globalen und langwelligen eintreffenden Strahlung auf der Turmspitze

**Comment by achristen** [comment]
4-Komponent Radiometer müsste eines in Paris verfügbar sein (Dana fragen) - wichtig auch Kabel. Prüfen ob ein Lüfter vorhanden ist, sonst einen bestellen.

**Comment by achristen** [comment]
PAR Sensoren wurden 2 Stück über ECOSENSE gekauft. Dirk Redepenning fragen, wo diese sind. Halterung bauen (Dirk).

**Comment by achristen** [comment]
Beheizbare Niederschlagswippe wurden schon über ECOSENSE gekauft. Dirk Redepenning fragen, wo diese ist. Halterung bauen (Dirk).

**Comment by achristen** [comment]
CS125 Sichtweiten und Niederschlagsart-Messgerät ("Present Weather Sensor") wurde über ECOSENSE gekauft. Logger-Programm ist in Hartheim integriert. Hier ist das Logger Programm: https://github.com/envmet/Hartheim/tree/master/HARTHM-CR1000X

**Comment by achristen** [comment]
5 Stück von Temperatur / Feuchte Sensoren sind Vorhanden, 2 Stück müssten noch gekauft werden. Die 5 Stück sind allerdings von Gregor "ausgeliehen", damit wir Parallelmessungen an den Stadtstation durchführen.

Klären ob: Gregor die weiterhin braucht und wir dort sogar mehr kaufen müssen.

**Comment by achristen** [comment]
Wind-Propeller wurde meines Wissens auch gekauft.  Dirk Redepenning fragen, wo diese sind. Halterung bauen (Dirk).

**Comment by achristen** [comment]
CR1000X Logger (du brauchst 3 Stück, 1 x dieses Issue Klimamessungen, 1 x Ersatz Eddy System Hartheim, 1 x Profil System)

**Comment by markus1203** [comment]
> CR1000X Logger (du brauchst 3 Stück, 1 x dieses Issue Klimamessungen, 1 x Ersatz Eddy System Hartheim, 1 x Profil System)

@achristen Brauche wir nicht 4 x CR1000X Logger? Da wir am Buchen- und Douglasien Turm Profile von Temperatur / Feuchte Sensoren aufbauen wollen?

**Comment by achristen** [comment]
Nein. Die Signale vom Buchen und Douglasienturm können über SDI-12 Leitungen zum Hauptturm geleitet werden. Wir brauchen dazu aber eine Kabeltrasse und 3-adriges Kabel. Die Kabel sind aber günstiger als ein Logger und alle Daten sind dann in der gleichen Datei. SDI-12 kann mehrere Km lang übertragen. Wir brauchen auch nur eine Leitung, da digitales Bus-System.

**Comment by achristen** [comment]
Man kann die zwei Türme auch in einem Y verbinden. Dann müssen nicht zwei Kabeltrassen gebaut werden, oder vom Buchen zum Douglasien und dann zum Hauptturm. Strom muss aber auch überall hin. Dazu mit Delon sprechen. Können wir Delon hier zufügen (auf GitHub?)

**Comment by markus1203** [comment]
> Beheizbare Niederschlagswippe wurden schon über ECOSENSE gekauft. Dirk Redepenning fragen, wo diese ist. Halterung bauen (Dirk).



> Beheizbare Niederschlagswippe wurden schon über ECOSENSE gekauft. Dirk Redepenning fragen, wo diese ist. Halterung bauen (Dirk).
Laut Dirk wurde noch keine beheizbare Niederschlagswippe für ECOSENSE gekauft

**Comment by markus1203** [comment]
> 5 Stück von Temperatur / Feuchte Sensoren sind Vorhanden, 2 Stück müssten noch gekauft werden. Die 5 Stück sind allerdings von Gregor "ausgeliehen", damit wir Parallelmessungen an den Stadtstation durchführen.
> 
> Klären ob: Gregor die weiterhin braucht und wir dort sogar mehr kaufen müssen.
> 
> Sollten alle auch Kalibriert oder verglichen werden. Taupunkt-Generator funktioniert wieder.
> 
> Angebot einholen inkl. Strahlungsschutz - entweder für 2 Stück oder für 11-12 Stück.

3 Stück hat Gregor mir schon gegeben, ein HygroVue verwendet Gregor noch und ein weiterer ist bei Dana in Paris.
Bei den 3 vorhandenen HygroVUEs fehlt allerdings das Kabel, bzw der Teil der an den Sensor angeschlossen wird wurde vom Kabel abgeschnitten

**Comment by achristen** [comment]
Jürgen Kreuzwieser am 10.08.2023:

> Lieber Andi,
> 
> Annette hat gesehen, dass auch die Wippe tatsächlich letztes Jahr schon aus Z1 bestellt wurde. Allerdings lief das unter Markus Weiler. 
> Da im Antrag steht, dass sie für "above-canopy measurements" vorgesehen ist, sollte die eigentlich für den Turm sein. Klärst du das mit Markus nochmal ab? 
> 
> Viele Grüße
> Jürgen
> 
>

**Comment by achristen** [comment]
Das heisst diese ist bei Markus Weiler. Kannst du ihm schreiben. Ich schicke dir die Email.

**Comment by markus1203** [comment]
Niederschalgswippe befindet sich momentan auf der Freifläche bis der Turm aufgebaut ist. 
Möglicherweise wäre es sinnvoll eine zweite für den Turm zu kaufen und die vorhandene auf der Freifläche zu lassen.

**Comment by markus1203** [comment]
> oder vom Buchen zum Douglasien und dann zum Hauptturm

diese Variante ist vermutlich am sinnvollsten

**Comment by markus1203** [comment]
Bilder wo die Türme gebaut werden sollen:

Hauptturm:
![signal-2023-09-21-125434_002](https://github.com/envmet/ECOSENSE/assets/60688751/b1e2a7fc-8dd7-4d42-8c07-86aafcd352bf)

Buchenturm:
![signal-2023-09-21-115317_003](https://github.com/envmet/ECOSENSE/assets/60688751/734f7bce-f24c-4fd9-866f-113918b98bb2)

Douglasienturm:
![signal-2023-09-21-115317_007](https://github.com/envmet/ECOSENSE/assets/60688751/e4cc969b-2748-48c5-9c07-3a0adf19f44f)

**Comment by markus1203** [comment]
Momentan habe ich 1 funktionieren CR1000X Logger und 1 defekten Logger.

Der Defekte kann möglicherweise mit Dirk repariert werden ("Evt. ist die Sicherung raus geflogen").

CR1000X Logger kosten momentan 2.328,00 €.
